fcrepo-stats-api/src/main/java/org/fcrepo/stats/api/StatsResults.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree.
+ */
+package org.fcrepo.stats.api;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * @author dbernstein
```

Comment:
Please add a description, and if the Stats object gets renamed, rename this accordingly since they are directly coupled